src/interactions/map_city/enterStoreMainArea.interaction.js

```diff
@@ -0,0 +1,31 @@
+import { displayDialogue } from '../../utils';
+import { sampleStoreInteraction, sampleArcadeInteraction } from './enterMapArcade.interactions';
```

Comment:
move your code from enterMapArcade.interactions to this file because of my comment to revert the enterMapArcade.interactions.js file